app/filters.js

```diff
@@ -127,5 +127,42 @@ module.exports = (env) => {
     return regExp.exec(providerAndCode)[1]
   }
 
+  filters.statusClass = (status) => {
+    switch (status) {
+      // States that use the default tag style
+      // - 'Enrolled'
+      // - 'Conditions met'
+      // - 'Completed'
+
+      // Application phases
+      case 'Submitted':
+        return 'govuk-tag--grey'
+      case 'Pending':
+        return 'govuk-tag--blue'
+      case 'Offer withdrawn':
+        return 'govuk-tag--red'
+      case 'Application withdrawn':
+        return 'govuk-tag--orange'
+      case 'Declined':
+        return 'govuk-tag--orange'
+      case 'Rejected':
+        return 'govuk-tag--red'
+      case 'Conditions not met':
```

Comment:
This one is currently dark blue on prod 😬

Reply:
Oh, I had it down as `Conditions met`. There are definitely some funky colours hanging around. I think we need to take a more holistic look.